package.json

```diff
@@ -24,6 +24,7 @@
     "express": "^4.16.3",
     "express-ejs-layouts": "^2.4.0",
     "faker": "^4.1.0",
+    "nodemon": "^1.17.4",
```

Comment:
Isso pertence ao `devDependencies`. Deveria ser instalado com `npm install --save-dev nodemon` 😉

Reply:
Outro ponto é: Porque adicionamos esse `faker`? Aonde ele é usado?

Reply:
Ele foi instalado junto com o express-js-layouts, no início a gente até usou mas depois vimos que não precisava. Dá pra tirar mesmo.